vimish-fold.el

```diff
@@ -415,6 +415,71 @@ This feature needs `avy' package."
            folds-before-point)))
       (message "No more folds before point"))))
 
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Folding marks
+
+(defcustom vimish-fold-marks '("{{{" . "}}}")
```

Comment:
I'm not against enabling this by default, but I think there should be a way to disable the feature. How about making it so if this value is `nil` then the feature is disabled?